io.github.Archeb.opentrace.yml

```diff
@@ -0,0 +1,71 @@
+app-id: io.github.Archeb.opentrace
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+separate-locales: false
+rename-icon: opentrace
+rename-desktop-file: opentrace.desktop
+command: OpenTrace
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
```

Comment:
why both?

Reply:
I'm pretty new in flatpak so that part is just copied from https://github.com/flathub/com.tencent.wemeet. Removed wayland now :)